packages_rs/nextclade/src/align/align.rs

```diff
@@ -46,7 +46,7 @@ pub fn align_nuc(
     );
   }
 
-  if ref_len + qry_len < (10 * params.seed_length) {
+  if ref_len + qry_len < (20 * params.kmer_length) {
     // for very short sequences, use full square
```

Comment:
According to Richard's recommendation, for short sequence check let's use `kmer_length` instead of removed `seed_length` and adjust the coefficient to roughly match the old value.